shared/actions/tracker.js

```diff
@@ -4,33 +4,24 @@ import Session from '../engine/session'
 import _ from 'lodash'
 import engine from '../engine'
 import openUrl from '../util/open-url'
```

Comment:
splitting imports / import types. moving private functions to use _. moving exports to the bottom